lib/fastlyctl/commands/token.rb

```diff
@@ -20,25 +20,27 @@ def token(action)
         scope = options[:scope]
         scope ||= "global"
 
-        say("You must login again to create tokens.")
+        say("You must authenticate again to create tokens.")
 
-        login_results = FastlyCTL::Fetcher.login
-
-        name = ask("What would you like to name your token?")
+        username = ask("Username: ")
+        password = ask("Password: ", :echo => false)
+        say("")
 
+        name = ask("What would you like to name your token? (enter here):")
         o = {
-          user: login_results[:user],
-          pass: login_results[:pass],
-          code: login_results[:code],
+          username: username,
+          password: password,
           scope: scope,
-          name: name
-        }
-
-        o[:services] = options[:services].split(",") if options[:services]
+          name: name || "fastlyctl_token"
+        }.compare_by_identity
 
+        options[:services].split(",").each do |v|
+          o["services[]"] = v
```

Comment:
Is this a typo?

Reply:
@stephenbasile so I tested several patterns to make it work, but looks like we need to explicitly set `services[]` as a key as well as set `compare_by_identity` for the options hash in order to set multiple keys with the same name (or, construct raw body strings manually). Otherwise, Typhoeus can't handle an array value well:
https://github.com/fastly/fastlyctl/blob/v1.0.16/lib/fastlyctl/commands/token.rb#L37

Our API documentation:
https://developer.fastly.com/reference/api/auth/
> **Services**
> Tokens are granted access to all services in an account by default. However, you can optionally restrict the service-level access of a token to one or more services. Do this by specifying an array in the POST `/tokens` action (e.g., `services[]=id1&services[]=id2`).

Reply:
@stephenbasile can I get 👍 on this?